src/api/auth/auth.ts

```diff
@@ -0,0 +1,24 @@
+// import { axiosInstance, obtainTokenApi, saveTokens } from "../api"
+
+// const LOGIN_URL = '';
+
+export interface UserAuthenticationData {
+  email: string;
+  password: string;
+}
+
+const loginUser = (data: UserAuthenticationData): Promise<void> => {
+  const { email, password } = data;
+
+  if (email !== 'test@test.com') throw new Error('Email does not exist!');
```

Comment:
let's remove this for now, we should have this handled normally by BE

Reply:
This is just for testing purposes, to show the errors on the login. Login error UI is still at work